types/nightwatch/index.d.ts

```diff
@@ -4147,26 +4199,26 @@ export interface ElementCommands {
      *   }
      * }
      *
-     * @see waitForElementPresent
+     * @see https://nightwatchjs.org/api/waitForElementNotPresent.html
      * @since v0.4.0
      */
     waitForElementNotPresent(
         selector: Definition,
         time?: number,
         poll?: number,
         abortOnFailure?: boolean,
-        callback?: (this: NightwatchAPI, result: NightwatchCallbackResult<void>) => void,
+        callback?: (this: NightwatchAPI, result: NightwatchCallbackResult<[ {ELEMENT_KEY: string }]>) => void,
```

Comment:
You are literally returning an object with a property called `ELEMENT_KEY`.

You probably want to use the existing `ELEMENT_KEY` const: `{ [ELEMENT_KEY]: string }`

Reply:
Maybe also create an additional type for this object, like
```ts
type ElementResult = { [ELEMENT_KEY]: string }
```